weave/trace_server/trace_server_interface.py

```diff
@@ -547,15 +547,38 @@ class TableCreateRes(BaseModel):
 
 
 class TableRowFilter(BaseModel):
-    row_digests: Optional[List[str]] = None
+    row_digests: Optional[List[str]] = Field(
```

Comment:
might be nice to show an example of what a digest actually looks like: 

`aonareimsvtl13apimtalpa4435rpmgnaemrpgmarltarstaorsnte134avrims`